### ISSUE TITLE
removeWrongOnInput doesn't track the changes in <select> just in <input>

### ISSUE THREAD

**Issue by imeckr** [Kinark/Materialize-stepper#85]
The current implementation of `removeWrongOnInput` only listens to changes in `<input>` but not in `<select>`.  I know the name is explicitly mentions `inputs` not `select`, but shouldn't that be the case as they are also kind of `input`?
https://github.com/Kinark/Materialize-stepper/blob/be100c4414fe18c32d19c232613bbdd297d2966e/src/js/mstepper.js#L351

**Comment by Kinark** [comment]
Yep, it definitely should be listening for select changes as well. Gonna fix it ;)

Or you can help me by submitting a PR to the develop branch :D
